hack/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+# $1 - Specifies distribution - RHEL7/CentOS7
+# $2 - Specifies MySQL version - 5.5
+
+# Array of all versions of MySQL
+declare -a VERSIONS=(5.5)
+
+OS=$1
+VERSION=$2
+
+
+function build_rhel {
+  mv Dockerfile Dockerfile.centos7
+  mv Dockerfile.rhel7 Dockerfile
+  trap "mv Dockerfile Dockerfile.rhel7 && mv Dockerfile.centos7 Dockerfile" RETURN
```

Comment:
good idea, can you make this change in our other STI repos?  it would be good to keep this logic identical across our repos.